src/containerapp/azext_containerapp/custom.py

```diff
@@ -1958,7 +1958,7 @@ def create_or_update_github_action(cmd,
                     return r
 
             raise ValidationError(
-                "Failed to find workflow file for Container App '{}' in .github/workflow folder for repo '{}'. ".format(name, repo) +
+                "Failed to find workflow file for Container App '{}' in .github/workflow folder for repo '{}'. ".format(name, repo) + "Please try again by re-running the command."
```

Comment:
Would we need to end this line by a "+" so that the string can be concatenated with the next error string?
Also, should we directly put the new error string with the previous one rather than concatenating it?

Something like:
```
"Failed to find workflow file for Container App '{}' in .github/workflow folder for repo '{}'. Please try again by re-running the command. ".format(name, repo) +
```